script/utils/viemScriptHelpers.ts

```diff
@@ -44,6 +44,9 @@ export const getViemChainForNetworkName = (networkName: string): Chain => {
         http: [network.rpcUrl],
       },
     },
+    contracts: {
+      multicall3: { address: network.multicallAddress as `0x${string}` },
```

Comment:
I think it's better to use `getAddress` as it ensures the address will be in the correct format, checksummed etc.

![image](https://github.com/user-attachments/assets/10e67a50-f7d9-439a-83dd-587baa155496)

Reply:
good point....added